api.ts

```diff
@@ -1108,6 +1110,25 @@ export interface SyncUpdateResultMarshaled<
  */
 export type GenericSyncUpdateResultMarshaled = SyncUpdateResultMarshaled<any, any, any>;
 
+/**
+ * Sets a limit on the number of permissions that can be returned for a single row
+ * in a call to {@link executeGetPermissions}.
+ *
+ * @hidden
```

Comment:
Did you want TODOs to unhide this and the next occurrence?

Reply:
I'm curious why this was marked resolved? Seems like a valid point?